### DISCUSSION TITLE
建议开发一个算法解题思路分享平台

### DISCUSSION THREAD

**Discussion by ghost** [zihaoyy/LeetCode#11] [Q&A]
建议开发一个算法解题思路分享的平台，这样可以让大家分享自己在刷算法题的时候分享自己的见解，毕竟一千个人心中有一千个哈姆雷特哈哈哈

**Comment by zihaoyy** [comment]
> 建议开发一个算法解题思路分享的平台，这样可以让大家分享自己在刷算法题的时候分享自己的见解，毕竟一千个人心中有一千个哈姆雷特哈哈哈

这个可以考虑一下！
